pkg/rainbow/option.go

```diff
@@ -22,13 +22,6 @@ type Option struct {
 }
 
 type Order struct {
-	// Px is a known abbreviation for "Price" used by many Centralized Exchanges
-	// such as in the FIX protocol: https://fiximate.fixtrading.org/legacy/en/FIX.5.0SP2/abbreviations.html
-	// Independently of the FIX protocol, "Px" is intensively used at Euronext.
-	// "Px" is also a French abbreviation for "Prix". :-)
-	// Rainbow uses "px" in the API (JSON)) but uses "price" on the front-end side.
-	Px float64 `json:"px"`
-
-	// Size is often used in lieu of the longer word "Quantity".
-	Size float64 `json:"size"`
```

Comment:
We should keep the documentation somewhere that justify the use of "px" and "size" in the API. If we delete these comment lines, where should we keep this documentation?

Reply:
We shoul do an external document for the API. The API is not a "backend only thing", so I don't think it's place for such a thing. A frontend dev should not have to go in the backend code to understand the API.